roles/openshift_health_checker/test/etcd_imagedata_size_test.py

```diff
@@ -56,7 +57,7 @@ def test_cannot_determine_available_mountpath(ansible_mounts, extra_words):
     with pytest.raises(OpenShiftCheckException) as excinfo:
         check.run()
 
-    for word in 'determine valid etcd mountpath'.split() + extra_words:
+    for word in ['Unable to determine mount point'] + extra_words:
```

Comment:
I see the trouble, and while splitting into words makes the test less sensitive to whitespace changes, it's still the case that changing the wording is likely to require a change to the tests; and at the same time, it makes the test a little less accurate - you could potentially match a similar nearby message that had all the right words. I did find one false positive like this. So, it was my intent.